index.html

```diff
@@ -1,2 +1,27 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link type="text/css" rel="stylesheet" href="css/normalize.css">
+    <link type="text/css" rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Monoton" rel="stylesheet">
+    <title>Paintmaster 3000</title>
+</head>
+<body>
+   <div class="container">
+       <h1>Paintmaster 3000</h1>
+       <aside>
+           <div class="button white"></div>
+           <div class="button orange"></div>
+           <div class="button yellow"></div>
+           <div class="button green"></div>
+           <div class="button blue"></div>
+           <div class="button pink"></div>
+           <div class="button eraser"></div>
+       </aside>
+       <main>
+           <div class="grid"></div>
```

Comment:
whats the reason for the div inside of the main, could the class of grid be put on main and do away with the extra layer of div?